setup.py

```diff
@@ -41,6 +41,19 @@
                 'distributed.http'],
       long_description=(open('README.rst').read() if os.path.exists('README.rst')
                         else ''),
+      classifiers=[
+        "Development Status :: 5 - Production/Stable",
```

Comment:
Seems fine to me.  This seems project seems stable relative to most other projects.